src/java/neroxis/transformer/MapTransformer.java

```diff
@@ -37,6 +37,11 @@
     private boolean reverseSide;
     private float angle;
     private SymmetrySettings symmetrySettings;
+    private int resize = -1;
+    private int mapSize = -1;
```

Comment:
If you are just going to default the values there is no need to set the value here. Just set it using an else in the argument parsing.

Reply:
The proper size can't be defaulted to at that point though.

Reply:
made flags